src/common/Types.ts

```diff
@@ -192,6 +192,11 @@ export interface ViewStyle extends ViewAndImageCommonStyle {
     // http://facebook.github.io/react-native/releases/0.30/docs/shadow-props.html (iOS only)
     // http://facebook.github.io/react-native/releases/0.30/docs/view.html#style (see elevation property)
     elevation?: number; // Android only
+    // Windows 10 RS3 supports acrylic brushes. In earlier versions these properties are ignored.
+    // The tint opacity can be set either with acrylicOpacity or with acrylicTintColor, e.g. #f002.
+    acrylicOpacity?: number; // UWP only; default = 1
```

Comment:
New style attributes should be added to the documentation. See Styles.md.

Also, consider renaming these to end in "UWP" since they're UWP specific. We have tried to do this where possible (e.g. "shouldRasterizeIOS"). React Native is somewhat inconsistent on this convention though, so ReactXP reflects this inconsistency.